src/main/java/org/opensearch/flowframework/util/ApiSpecFetcher.java

```diff
@@ -87,23 +89,13 @@ public static boolean compareRequiredFields(List<String> requiredEnumParams, Str
     }
 
     private static Content getContent(RestRequest.Method method, PathItem pathItem) throws IllegalArgumentException, ApiSpecParseException {
-        Operation operation;
-        switch (method) {
-            case POST:
-                operation = pathItem.getPost();
-                break;
-            case GET:
-                operation = pathItem.getGet();
-                break;
-            case PUT:
-                operation = pathItem.getPut();
-                break;
-            case DELETE:
-                operation = pathItem.getDelete();
-                break;
-            default:
-                throw new IllegalArgumentException("Unsupported HTTP method: " + method);
-        }
+        Operation operation = switch (method) {
+            case POST -> pathItem.getPost();
+            case GET -> pathItem.getGet();
+            case PUT -> pathItem.getPut();
+            case DELETE -> pathItem.getDelete();
+            default -> throw new IllegalArgumentException("Unsupported HTTP method: " + method);
+        };
```

Comment:
While this is a great change for JDK 14+ we still support JDK 11 on the 2.x backport.  So until we are on 3.x as the main release version we'd like to keep the older JDK compatibility.  So can you please revert this change for now?

Reply:
Sure. Just saw the current version was at 21 and thought of refactoring. Let me revert this.

Reply:
Yeah, we're JDK21 minimum on 3.x branch but 3.0.0 release is still a couple months out so we are regularly backporting to 2.x.  It's a lot easier if there aren't major code changes.   Once we're tracking 3.x regularly and only doing bugfixes on 2.x we can do more JDK21+ optimizations.

Reply:
Got it!